core/native/src/main/scala/zio/UpdateRuntimeFlagsWithinPlatformSpecific.scala

```diff
@@ -0,0 +1,8 @@
+package zio
+
+// This is split because `java.util.function.IntFunction` isn't available in Scala Native yet.
+// Once it's available, this should be merged together with the `js-jvm` variant back to `shared`.
+// https://github.com/scala-native/scala-native/pull/3127
+private[zio] trait UpdateRuntimeFlagsWithinPlatformSpecific {
+  type RuntimeFlagsFunc[A] = Int => A
```

Comment:
btw, do we need to mark it as `private[zio]` too?

Reply:
I think if the trait is package private and the only object that extends it is package private there is no need for the members to be package private as well.